AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Service/Helpers/ListDevicesOutputResponse+Helper.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import Amplify
+import AWSCognitoIdentityProvider
+import AWSPluginsCore
+
+extension CognitoIdentityProviderClientTypes.DeviceType {
+    
+    func toAWSAuthDevice() -> AuthDevice {
+        let id = deviceKey ?? ""
+        let name = ""
```

Comment:
We do not have a corresponding value for device name?

Reply:
yes it seems like it. didn't see a device name field
https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_ListDevices.html#API_ListDevices_ResponseSyntax